### PR TITLE
Spectra Support - Added custom fonts support for Spectra plugin

### PR DESCRIPTION
**Pull request by sushmak02**
Fonts added with the custom fonts plugin are not loading in the Spectra editor.

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

**Comment by imnavanath** [comment]
Assigned testing task to @SimranChougule1 https://brainstormforce.atlassian.net/browse/AST-2068.
@sushmak02 by the time can you please add changelog for the same in readme. 
**Note:** Don't merge this PR directly, FYI, once it gets merged update will be release so release checkpoints needs to follow here.

**Comment by sushmak02** [comment]
@imnavanath I have done the requested changes and also updated the version number so we can release by merging this PR.

@SimranChougule1 Has already tested this in Spectra so, please get this reviewed quickly.

**Review by imnavanath — APPROVED** [review]

**Pull request merged**
